pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -479,6 +480,69 @@ def _build_circbox(instr: Instruction, circuit: QuantumCircuit) -> CircBox:
     return CircBox(subc)
 
 
+# Used for handling of IfElseOp
+# docs -> https://docs.quantum.ibm.com/api/qiskit/qiskit.circuit.IfElseOp
+# Examples -> https://docs.quantum.ibm.com/guides/classical-feedforward-and-control-flow
+def _pytket_boxes_from_ifelseop(
+    if_else_op: IfElseOp, qregs: list[QuantumRegister], cregs: list[ClassicalRegister]
+) -> tuple[CircBox, Optional[CircBox]]:
```

Comment:
I'm not so pleased with this `Optional` return type in the function signature. This is to deal with the case when the `IfElseOp` only has a `true_body` rather than a `true_body` and a `false_body`.

There's probably a cleaner way to deal with this.